components/director/internal/destinationfetchersvc/service.go

```diff
@@ -99,57 +101,61 @@ func (d *DestinationService) generateClientBySubdomainLabel(ctx context.Context,
 
 	subdomain, ok := subdomainLabel.Value.(string)
 	if !ok {
-		log.C(ctx).Errorf("cannot cast label value as a string")
 		return nil, errors.New("cannot cast label value as a string")
 	}
 
 	region, ok := regionLabel.Value.(string)
 	if !ok {
-		log.C(ctx).Errorf("cannot cast label value as a string")
 		return nil, errors.New("cannot cast label value as a string")
 	}
 
 	instanceConfig, ok := d.DestinationsConfig.RegionToInstanceConfig[region]
 	if !ok {
-		log.C(ctx).Errorf("No destination instance credentials found for region '%s'", region)
-		return nil, errors.New(fmt.Sprintf("No destination instance credentials found for region '%s'", region))
+		return nil, fmt.Errorf("no destination instance credentials found for region '%s'", region)
```

Comment:
The errors won't be logged in case of a force sync through the API (only in the cronjob)?
Log the error in (handler.go) SyncTenantDestinations